pyvista/plotting/widgets.py

```diff
@@ -1405,10 +1446,18 @@ def add_mesh_isovalue(
         The isovalue mesh is saved to the ``.isovalue_meshes``
         attribute on the plotter.
 
+        .. warning::
+            This will not work with :class:`pyvista.PointSet` as
+            creating an isovalue is a dimension reducing operation
+            on the geometry and point clouds are zero dimensional.
+            This will similarly fail for point clouds in
+            :class:`pyvista.PointSet`.
```

Comment:
Is `PointSet` mentioned here twice, was did this second one mean to be polydata?

Reply:
Crap! Meant PolyData in the second